OpenSim/Simulation/Model/GeometryPath.h

```diff
@@ -85,7 +87,9 @@ OpenSim_DECLARE_CONCRETE_OBJECT(GeometryPath, ModelComponent);
     //--------------------------------------------------------------------------
 public:
     GeometryPath();
-    virtual ~GeometryPath();
+#ifndef SWIG
+    ~GeometryPath() override = default;
```

Comment:
Do you think it will be a problem if there is no destructor for SWIG to wrap? Or does that happen automatically somehow? You could avoid this by writing `~GeometryPath() override {}`.